MonoGame.Framework/Graphics/GraphicsDevice.OpenGL.cs

```diff
@@ -88,11 +88,15 @@ internal void SetVertexAttributeArray(bool[] attrs)
             }
         }
 
-        private void PlatformSetup()
+       private void PlatformSetup()
         {
 #if WINDOWS || LINUX || ANGLE
             GraphicsMode mode = GraphicsMode.Default;
-            var wnd = (Game.Instance.Window as OpenTKGameWindow).Window.WindowInfo;
+
+            var wnd = OpenTK.Platform.Utilities.CreateWindowsWindowInfo(PresentationParameters.DeviceWindowHandle);
```

Comment:
Oh, sorry, didn't read the bellow comments. That does sound like a good plan, I'll try and setup a PR for it tomorrow after school. Also later on, what do you think of setting up MonoGame WinForm Control and maybe Gtk Widget projects in the MonoGame repository?